packages/graph/hash_graph/lib/graph/src/store/postgres/mod.rs

```diff
@@ -803,22 +803,26 @@ where
                 .change_context(InsertionError)?;
         }
 
+        // TODO: should we check that the `link_entity_type_ref` is a link entity type?
+        // TODO: Use an inheritance table for `inherits_from`
+        //   see https://app.asana.com/0/1202805690238892/1203277018227719/f
```

Comment:
```suggestion
        // TODO: should we check that the `link_entity_type_ref` is a link entity type?
        //   see https://app.asana.com/0/1202805690238892/1203277018227719/f
```

I don't think we necessarily want an inheritance table